modules/serverless-installing-service-mesh-member-roll.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// serverless/installing-openshift-serverless.adoc
+
+[id="installing-service-mesh-member-roll_{context}"]
+= Installing a ServiceMeshMemberRoll
+
+If Service Mesh is configured for multi-tenancy, you should have a Service Mesh Member Roll for the control plane namespace. For applications to make use of the deployed control plane and ingress, their namespaces need to be part of a member roll. 
```

Comment:
AFAIK, I'm not sure you can configure Service Mesh without multi-tenancy any more? I guess we'd have to verify this with the Istio team, but that is my understanding.

Reply:
@alanfx Should the 'If" be removed and this made into an imperative ( i.e You must have a Service Mesh Member Roll...)